packages/apollo-cli/src/__mocks__/localfs.ts

```diff
@@ -0,0 +1,12 @@
+import {Volume, createFsFromVolume} from 'memfs';
```

Comment:
This is the main part of how fs mocking works. Instead of importing `fs` everywhere, we import `localfs` which points to `fs` when regularly running or the in-memory volume in unit tests.